src/Cipher/Autokey/AutokeyStream.tsx

```diff
@@ -0,0 +1,136 @@
+import * as React from 'react';
+import { Button, ButtonGroup, ButtonToolbar, FormControl, ToggleButton, ToggleButtonGroup } from 'react-bootstrap';
+import LocalStorage from 'Data/LocalStorage';
+import { AutoKeyString } from 'puzzle-lib';
+import './AutokeyStream.css';
+
+type Props = {};
+type State = {
+  conversion: number,
+  key: string,
+  output: string,
+  text: string,
+};
+
+type SavedState = {
+  conversion: number,
+  key: string,
+  text: string,
+};
+
+class AutokeyStream extends React.Component<Props, State> {
```

Comment:
This is 99% copy/pasted from VigenereStream.  I'm thinking of refactoring them into a common KeyedCipherBaseStream class that takes a decoder object.  Any objections/reasons I shouldn't?

Reply:
I don't really see why not, should probably introduce a new interface/base class in puzzle-lib to go with it.

Reply:
Took care of puzzle-lib: https://github.com/kfarnung/puzzle-lib/pull/25